docs/diagrams/ParserClasses.puml

```diff
@@ -21,10 +21,10 @@ Class Prefix
 Class HiddenOutside #FFFFFF
 HiddenOutside ..> AddressBookParser
 
-AddressBookParser .down.> XYZCommandParser: <<create>>
+AddressBookParser .down.> XYZCommandParser: create >
 
-XYZCommandParser ..> XYZCommand : <<create>>
-AddressBookParser ..> Command : <<use>>
+XYZCommandParser ..> XYZCommand : create >
+AddressBookParser ..> Command : use >
```

Comment:
What do you think about rephrasing these to "creates" and "uses"? Looks like it'd be more in line with the examples in the textbook. Same for `LogicClassDiagram`!

<img width="739" alt="image" src="https://github.com/user-attachments/assets/19bd62cc-1752-45b4-81e9-3fc14f74a33b">